packages/angular-material/src/other/table.renderer.ts

```diff
@@ -48,12 +48,33 @@ import {
 @Component({
   selector: 'TableRenderer',
   template: `
+
     <table
       mat-table
       [dataSource]="data"
       class="mat-elevation-z8"
       [trackBy]="trackElement"
     >
+      <ng-container matColumnDef="action">
+        <tr>
+          <th mat-header-cell *matHeaderCellDef>
+            <button mat-icon-button color="primary" (click)="add()" [disabled]="!isEnabled()" title="Add"><mat-icon>add</mat-icon></button>
+          </th>
+        </tr>
+        <tr>
+          <td mat-cell *matCellDef="let row; let i = index" style="width:20%">
```

Comment:
I think we don't need the explicit width declaration here. Is there a reason that you added this?
```suggestion
          <td mat-cell *matCellDef="let row; let i = index">
```

Reply:
@lucas-koehler is it okay to add something like this? 
```
  styles: [
      'table {width: 100%;}',
      '.cdk-column-action { width: 5%}']
      ```

I think material is putting some auto width on columns, I wanted to shrink it. so cell is taking only small fraction of a row.

Reply:
Seems fine to me. I tested it in the example app and it seems to work well :)